conda_build/main_build.py

```diff
@@ -255,6 +255,7 @@ def execute(args, parser):
     import shutil
     import tarfile
     import tempfile
+    from os import makedirs
```

Comment:
I have no idea why main_build.py has so weird imports, but I have just followed the style.